src/qoi/src/rayfire_mesh.C

```diff
@@ -40,7 +40,7 @@ namespace GRINS
 {
   RayfireMesh::RayfireMesh(libMesh::Point& origin, libMesh::Real theta, libMesh::Real phi) :
     _dim(3),
-    _origin(origin),
+    _origin(libMesh::Point(origin)),
```

Comment:
Huh? You should be able to just pass origin and then `_origin` will be created from the copy constructor of `origin`. That needlessly introduces another copy construction.